mantle/kola/tests/misc/raid.go

```diff
@@ -57,52 +56,6 @@ var (
         wipe_filesystem: true`)
 )
 
-func init() {
-	register.RegisterTest(&register.Test{
-		// This test needs additional disks which is only supported on qemu since Ignition
-		// does not support deleting partitions without wiping the partition table and the
-		// disk doesn't have room for new partitions.
-		// TODO(ajeddeloh): change this to delete partition 9 and replace it with 9 and 10
-		// once Ignition supports it.
-		Run:         RootOnRaid,
-		ClusterSize: 0,
-		Platforms:   []string{"qemu"},
-		Name:        "cl.disk.raid.root",
-		Distros:     []string{"cl"},
-	})
-	register.RegisterTest(&register.Test{
-		Run:         DataOnRaid,
-		ClusterSize: 1,
-		Name:        "cl.disk.raid.data",
-		UserData: conf.ContainerLinuxConfig(`storage:
```

Comment:
I don't think it makes sense to keep the code but drop the config expected by the code.  If we're going to keep the code, we should keep this stuff too in some form, and just disable it.

Reply:
The code is required in another place so maybe I can move it there. Will have to double check that we have RAID tests for FOCS.

Reply:
Looks like we have RAID tests elsewhere for FCOS so I moved the code there and removed everything else from this file.